src/css/components/_control-bar.scss

```diff
@@ -29,9 +29,9 @@
   @include transition($trans);
 }
 
-.video-js.vjs-controls-disabled .vjs-control-bar,
-.video-js.vjs-using-native-controls .vjs-control-bar,
-.video-js.vjs-error .vjs-control-bar {
+.video-js.video-js.video-js.vjs-controls-disabled .vjs-control-bar,
+.video-js.video-js.video-js.vjs-using-native-controls .vjs-control-bar,
+.video-js.video-js.video-js.vjs-error .vjs-control-bar {
```

Comment:
I get the limitation here, but I don't think we should say this is an ok thing to do in the stylesheet. It's gonna result in uglier and uglier selectors, and I think we can do better.

- I'd prefer `important` here over a triple class name. It is after all important that the controls be hidden when the controls are disabled.

- We could try lowering some of the selector strengths and using strategic ordering over increasing class names.

- We could hide the controls in additional ways to `display`, so you have to do more than just display:table to override it.